src/components/FSHControls.js

```diff
@@ -164,9 +169,23 @@ export default function FSHControls(props) {
     props.onGoFSHClick('', true);
     const gofshInputStrings = props.gofshText.map((def) => def.def).filter((d) => d);
     const parsedDependencies = dependencies === '' ? [] : dependencies.split(',');
+
+    // Create small ImplementationGuide resource to send canonical and version information
+    if (canonical || version) {
+      const igResource = {
+        resourceType: 'ImplementationGuide',
+        fhirVersion: ['4.0.1'],
+        ...(canonical && { url: canonical }),
```

Comment:
Technically, I think that the IG `url` is not the canonical, but is _based_ on the canonical.  So it really should be something more like: ``${canonical}/ImplementationGuide/1`` (which assumes the IG JSON has id `1` - which you might as well also add).

Reply:
I should have thought about this, so thanks for noticing! I'm changing it to be the pattern you described, which is what SUSHI uses to build the `url` in the IG resource [here](https://github.com/FHIR/sushi/blob/master/src/ig/IGExporter.ts#L133), and adding the `id` as well.

Reply:
Addressed here: 8873011